src/tools/perf/perftest.h

```diff
@@ -53,4 +60,10 @@ void print_progress(void *UCS_V_UNUSED rte_group,
                     const ucx_perf_result_t *result, void *arg,
                     const char *extra_info, int final, int is_multi_thread);
 
+void backup_local_params(const perftest_params_t *params,
```

Comment:
maybe add as static inline in `perftest_context.h`?

Reply:
done